README.md

```diff
@@ -1,12 +1,53 @@
-## How to set navigation
-- Create a directory hierarchy in `src/pages/config.md`
+# Adobe Connect SDK Documentation
 
-## Launching a deploy
-- Go to Actions > Deployment > Run workflow
+Adobe Connect SDK documentation site deployed to EDS.
+The production address is https://developer.adobe.com/adobe-connect-sdk/
```

Comment:
Production always matches on what is on the `main` branch so that should be noted

Reply:
see below